extensions/widgets/treeview/notes/feature-auto_expand.md

```diff
@@ -0,0 +1,10 @@
+# Properties
+
+The tree view widget now has the ability to automatically expand to reveal
+a row when it is selected. If **scrollHilitedElementIntoView** is not `true`
+then the scroll position will be adjusted to maintain the currently visible
+top row.
+
+The tree view fold state can now be reset in two ways. To set the array
```

Comment:
@bwmilby My first inclination was to think this property allowed control over the folded state over the entire tree but it seems it just clears the folding. I'm wondering if this should be a boolean property. Either everything is folded or it's not. So:
```
set the folded of widget "tree" to true -- folds it all
get the folded of widget "tree" -- returns true if it's entirely folded
```
The only issue is what to do with `set the folded of widget "tree" to false` if there isn't currently an unfolded node... perhaps just unfold the first node in sort order?

Reply:
This is actually kind of a hack to get around the lack of directly calling handlers.  So I had to encode what would be a parameter into the property name.  The reasoning is to allow the array data to be replace and also reset the fold state in a single operation.  This prevents the applying of the fold state and the related calculations when the end desire is to have it folded.  The bug report was requesting the ability to reset the fold state when setting the array data.

Reply:
Possibly `arrayDataAndFoldAll` for the property name?